### ISSUE TITLE
Time is wrong on log filename

### ISSUE THREAD

**Issue by jazztickets** [flightlessmango/MangoHud#176]
Not sure why you're adding 1 to hour, min and sec.

It should be:
```
string date = to_string(log_time->tm_year + 1900) + "-" +
              to_string(1 + log_time->tm_mon) + "-" +
              to_string(log_time->tm_mday) + "_" +
              to_string(log_time->tm_hour) + "-" +
              to_string(log_time->tm_min) + "-" +
              to_string(log_time->tm_sec);
```
http://www.cplusplus.com/reference/ctime/tm/

**Comment by jazztickets** [comment]
Better yet, just use std::put_time so you get padded zeroes:

```
time_t now_log = time(0);
tm *log_time = localtime(&now_log);
std::ostringstream os;
os << std::put_time(log_time, "%Y-%m-%d_%H-%M-%S");
string date = os.str();
```

**Comment by flightlessmango** [comment]
fixed here 096fb626a328b6dd8d6ca9179d86a133a9e4d67b
